config/config.go

```diff
@@ -123,6 +125,10 @@ type Config struct {
 	lock        sync.RWMutex
 	AcceptAddrs []string `json:"acceptAddrs"`
 	AdminAddrs  []string `json:"adminAddrs"`
+
+	// nconnect network
+	NodeName       string `json:"nodeName,omitempty" long:"node-name" description:"Node name, will be used as nConnect network node name"`
+	ManagerAddress string `json:"managerAddress,omitempty" long:"manager-address" description:"Manager address, will be used as nConnect network manager address"`
```

Comment:
(network member only) Manager address to connect to when joining a network

Reply:
Yes, modified.